jargit.py

```diff
@@ -180,7 +182,7 @@ def normal_print(*args, **kwargs):
             # to see if the server false to get started
             verbose_or_normal_print('-' * width)
             verbose_or_normal_print('Waiting on Service Status'.center(width), end='\r')
-            sleep(5.0)
+            sleep(WAIT_TIME_SECONDS)
```

Comment:
I reduced the wait time down to two seconds from five to speed things up a bit. a small sleep is needed to allow the service time to start up so the status displayed will actually being meaningful. I'm considering making the wait time an argument so that the user can override the default wait time with a command line argument.